.github/workflows/ruby_head_specs.yml

```
@@ -24,13 +24,20 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
-      
-      - name: Set up Ruby (head)
+
+      # HACK: 無理やり.ruby-versionを書き換えつつ、Gemfileの Ruby versionの指定を消す
+      - run: |
```

actionlint (1.7.4)

property "custom-ruby-version" is not defined in object type {}

(expression)